src/ts/person_list.ts

```diff
@@ -0,0 +1,87 @@
+import $ from "jquery";
+import Person from "./person";
+import { wheelSync } from "./wheel_event";
+
+class PersonList {
+  private _people: Person[] = [];
+  private static _instance: PersonList;
+
+  constructor(...people: Person[]) {
```

Comment:
Is there a way to do 
```suggestion
  constructor(...people: Person[] | string[]) {
```
If so, remove `fromNames`.